pkgs/applications/window-managers/sway/lock-effects.nix

```diff
@@ -6,13 +6,13 @@
 
 stdenv.mkDerivation rec {
   pname = "swaylock-effects";
-  version = "v1.6-1";
+  version = "v1.6-2";
```

Comment:
Just a note: We don’t usually include the `v` in `version`, but that’s not changing on this patch so we don’t need to block on it.

Reply:
Oh yeah, I should have seen this. Thanks